File: Java/Lab06/Q1.java
import java.util.*;
import java.util.Scanner;
class Array {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        System.out.println("Enter the Number the Size of the Array:");
        int n = scan.nextInt();
        int[] arr = new int[n];
        System.out.println("Enter the Elements " + n + " numbers:");
        for(int i = 0; i < n; i++){
            System.out.println("Enter the Element " + (i+1)+ ":");
            arr[i] = scan.nextInt();
        }
        System.out.println("Entered Array");
        for(int i = 0; i < n; i++){
            System.out.print(arr[i] + " ");
        }
        System.out.println();
        Arrays.sort(arr); 
        System.out.println("Sorted Array");
        for(int i = 0; i < n; i++){
            System.out.print(arr[i] + " ");
        }
    }
}

File: DS Lab/DS Lab 1/Q1.c
#include <stdio.h>
int n;
int main()
{
    int arr[5] = {3, 9, 6, 2, 5};
    int element, position;
    printf("Enter the Element & Position:\n");
    scanf("%d %d", &element, &position);
    for (int i = 5; i >= position - 1; i--)
    {
        arr[i + 1] = arr[i];
    }
    arr[position - 1] = element;
    n++;
    printf("Elements of an Array:\n");
    for (int i = 0; i < 5; i++)
    {
        printf("%d\t", arr[i]);
    }
}
